pykov.py

```diff
@@ -1085,7 +1085,13 @@ def walk(self, steps, start=None, stop=None):
         ['B', 'A', 'A', 'A', 'A', 'A', 'B']
         """
         if start is None:
-            start = self.steady().choose()
+            steady = self.steady()
```

Comment:
I was referring to the documentation of `.walk()`, and, yes, I agree that documenting more the non-ergodic case is a good idea.